fontforgeexe/charinfo.c

```diff
@@ -3653,6 +3659,73 @@ static int CI_DefLCChange(GGadget *g, GEvent *e) {
 return( true );
 }
 
+// Turn the user's entered decomposition into the format returned by SFGetAlternate
+unichar_t* CI_ParseUserDecomposition(char* inp) {
+    unsigned long len = strlen(inp);
+    char* end;
+    unichar_t* out = malloc(len*sizeof(unichar_t)+1);
```

Comment:
`(len+1)*sizeof(unichar_t)`

Reply:
Fixed in https://github.com/fontforge/fontforge/pull/3645/commits/a3135465fd4aa05d984d7f06870d5b13601fb71b (I hope)